src/layout/layout_cache.cpp

```
@@ -38,8 +38,8 @@ namespace moth_ui {
 
     std::shared_ptr<Layout> LayoutCache::LoadLayout(std::string_view name) {
         std::string const filename = fmt::format("{}/{}.json", m_root, name);
```

Clang (14.0.6)

[error] variable 'filename' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)